### DISCUSSION TITLE
Add a Random finding read button

### DISCUSSION THREAD

**Discussion by V1C70RYG0D** [solodit/feedback#6] [New Feature Requests]
add a button to randomly read findings, 
it can be integrated with search option to search random finding in any category or from all cateogries and also an filter out the completed ones
so that i can randomly read findings which i hadnt completed yet , be its a specific to any topic

Thank you so much for your effort
and thanks a lot you fullfilled my previous request of Completed button <3

**Comment by hans-cyfrin** [comment]
Hi, @V1C70RYG0D,
We appreciate your suggestion. 
Currently, we're evaluating whether this feature would benefit a significant number of users. 
Rest assured, we'll keep this discussion open and will consider it further later. 
Thanks again for your input.

**Reply by Grimxjoke** [reply]
I personally think that it's against the Solodit's functionality. The main goal of this tool (imo) is to search specific features and look for their limitations instead of randomly looking at issues.
